components/PickerBase/PickerBase.behavior.comp.cy.js

```diff
@@ -226,4 +226,111 @@ describe('Test the PickerBase component behavior', () => {
         });
     });
   });
-});
+
+  it('should initially show "✅ All" when no items are selected', () => {
```

Comment:
This test is redundant with the similar one in `content`.  It is checking the _content_ of the component when it is rendered.  So it belongs in `content` so this one should be deleted and the one in `content` should be kept.